automation/TaskManager.py

```diff
@@ -469,25 +470,27 @@ def _issue_command(
             browser.command_queue.put(command)
 
             # received reply from BrowserManager, either success or failure
-            error_text = None
-            tb = None
-            status = None
+
+            status = Status()
+
             try:
-                status = browser.status_queue.get(True, browser.current_timeout)
+                status.raw_tuple(
+                    browser.status_queue.get(True, browser.current_timeout)
+                )
             except EmptyQueue:
                 command_status = "timeout"
                 self.logger.info(
                     "BROWSER %i: Timeout while executing command, %s, killing "
                     "browser manager" % (browser.browser_id, repr(command))
                 )
 
-            if status is None:
+            if status.raw_tuple is None:
```

Comment:
This checks if the method exists on the object. Is this what you wanted to do?

Reply:
Yes, I needed to check if the function exists, that way I'd know that Status was successfully instantiated. Though it was used for debugging, I forgot to remove that